src/renderer/component/mapping/new-mapping-destination.js

```diff
@@ -31,6 +33,7 @@ const NewMappingDestination = (props) => {
 
   return <div className="mapping-destination">
     <h1>Enter destination</h1>
+    { errors.length ? <Errors errors={ errors }/> : '' }
```

Comment:
`Errors` should include an empty check so you're not duplicating this logic every time.

Reply:
@tomitm Sounds good.

To confirm: we're talking about moving this logic into the `Errors` component itself?

Reply:
Yep, just a simple `if (!errors.length) return null` within `Errors` should suffice. Then you don't need to worry about it in other components.

Reply:
Sounds good. Will drop that in this evening.